samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnContainerRunner.java

```diff
@@ -190,13 +196,24 @@ private void startContainer(Path packagePath,
       throw new SamzaContainerLaunchException("IO Exception when writing credentials to output buffer");
     }
 
+    Map<String, LocalResource> localResourceMap = new HashMap<>();
```

Comment:
the old yarn.package.path does not fit in the general LocalizerResourceConfig and LocalizerResourceMapper. As we would like to see if we can deprecate this one in favor of using yarn.resources.package.path which follows the general localizer resource config, I prefer not to hack  LocalizerResourceConfig and LocalizerResourceMapper to support this old config.